### ISSUE TITLE
Results on Verified

### ISSUE THREAD

**Issue by zyone1991** [logic-star-ai/swt-bench#9]
### Describe the issue

Hi,

Are you planning to release the evaluation results over SWTbench-Verified? I think adding this would help a lot, given that the verified dataset is more reliable.

**Comment by nielstron** [comment]
We have added the results of the top-performing methods on SWT-Lite to SWT-Verified in #18 

We will further report results of new methods as rolling releases.

**Comment by zyone1991** [comment]
Thank you. Do you have any the statistics for the gold evaluation on Verified, such as the number of resolved instances?

**Comment by nielstron** [comment]
Certainly. We ran gold verified 5 times and filtered out all instances that failed at least once (as we did with SWT-Bench Lite), resulting in a total of 433 instances for SWT-Verified. Th 67 failing instances are listed here:
https://github.com/logic-star-ai/swt-bench/blob/master/dataset/filter_cases_verified.txt

The BM25 extended dataset can be found on huggingface: https://huggingface.co/datasets/nmuendler/SWT-Bench_Verified_bm25_27k_zsp/

Below are the coverage and detailed test results for SWT-Verified

| Method                             | validate-verified-gold |
|------------------------------------|--------------------------|
| Applicability (W)                  | 100.0                    |
| Success Rate (S)                   | 100.0                    |
| F->X                               | 100.0                    |
| F->P                               | 100.0                    |
| P->P                               | 15.011547344110854       |
| Coverage Delta (Δᵃˡˡ)              | 69.12766689735216        |

**Comment by zyone1991** [comment]
If I understand correctly, future submissions on Verified would be evaluated based on the instances you list above?

**Comment by nielstron** [comment]
Yes that is correct. All submissions listed on the leaderboard are evaluated on the 433 instances as detailed above.

**Comment by nielstron** [comment]
I have now added these numbers and details to the main README
